opensrp-app/src/main/java/org/smartregister/view/fragment/SecuredNativeSmartRegisterFragment.java

```diff
@@ -439,8 +443,14 @@ private int getCurrentPageCount() {
         }
 
         public void refresh() {
+            int textSize = getResources().getDimensionPixelSize(R.dimen.pagination_page_info_size);
+            int startIndex = getResources().getString(R.string.str_page_info).indexOf("{");
```

Comment:
Any reason for this additional logic?

Reply:
https://github.com/OpenSRP/opensrp-client-core/pull/715#discussion_r551821442

Reply:
https://github.com/OpenSRP/opensrp-client-core/pull/715#discussion_r551864293